bin/helpers/constants.js

```diff
@@ -120,15 +120,19 @@ const validationMessages = {
   EMPTY_RUN_SETTINGS: "Empty run settings",
   EMPTY_CYPRESS_PROJ_DIR:
     "cypress_proj_dir is not set in run_settings. See https://www.browserstack.com/docs/automate/cypress/sample-tutorial to learn more.",
+  EMPTY_CYPRESS_CONFIG_FILE: 
+    "cypress_config_file is not set in run_settings. See https://www.browserstack.com/docs/automate/cypress/sample-tutorial to learn more.",
```

Comment:
Can we give this docs link: https://www.browserstack.com/docs/automate/cypress/specs-to-run

@pranavj1001

Reply:
Thanks for the comment, however, I've changed the link to https://www.browserstack.com/docs/automate/cypress/configuration-file